packages/image_picker/image_picker/README.md

```diff
@@ -60,7 +60,7 @@ class _MyHomePageState extends State<MyHomePage> {
             : Image.file(_image),
       ),
       floatingActionButton: FloatingActionButton(
-        onPressed: getImage,
+        onPressed: getImage(),
```

Comment:
This would call the getImage function when you're creating the FloatingActionButton, and not as an 'onPressed' handler, wouldn't it?

Reply:
You're right, I just double checked the code and I thought it was calling `onPressed: () => getImage` which was causing an issue when I tried to recreate it in my app.

Reply:
Ahhh gotcha! Thanks!